src/analytics.js

```diff
@@ -76,21 +76,13 @@ function sendTrackingLogEvent(eventName, properties) {
 }
 
 /**
- * Send identify call to Segment, using userId from authApiClient.
+ * Send identify call to Segment.
+ * @param userId
  * @param traits (optional)
  */
-function identifyAuthenticatedUser(traits) {
-  const apiClient = getAuthApiClient();
-  const authState = apiClient.getAuthenticationState();
```

Comment:
@douglashall: I think this change is good and solves the problem.  Thank you.

The point I was trying to make yesterday is that we now know that `apiClient.getAuthenticationState()` is unreliable to call from outside the apiClient, because it may sometimes provide nothing, and that could cause very-difficult-to-find-again issues.

Could you document the method with a comment like the one below, and find a way to keep it internal only?
```
// getAuthenticationState() is not safe to call by anyone other than the authClient, because in certain cases the token expires and no longer exists immediately after the authClient used it.
```